src/Datadog.Trace.ClrProfiler.Managed/Integrations/Caching/Interception.cs

```diff
@@ -23,17 +24,22 @@ internal static Type[] TypeArray(params object[] objectsToCheck)
             return types;
         }
 
+        internal static Type[] Types(params Type[] types)
```

Comment:
I like this!

Reply:
I have mixed feelings. I like the idea of it at first, but then I realize it's just a non-standard way of creating an array. It doesn't actually save on typing or anything, and it adds an extra method call that can't be inlined. Note: I'm not asking you to change it in this PR.

For comparison:
```csharp
Type[] types = {arg1, arg2, ...};
var types = new[] {arg1, arg2, ...};
var types = Interception.Types(arg1, arg2, ...);
```